manifests/install.pp

```diff
@@ -6,11 +6,12 @@
 
   if ! defined(User[$aptly::user]) {
     user { $aptly::user:
-      ensure  => present,
-      uid     => $aptly::uid,
-      gid     => $aptly::group,
-      shell   => '/bin/bash',
-      require => Group[$aptly::group],
+      ensure     => present,
+      uid        => $aptly::uid,
+      gid        => $aptly::group,
+      shell      => '/bin/bash',
+      managehome => true
```

Comment:
Adding that option should be an option (the shell should have been one too). Have service accounts with a home directory is not really a best practice IMO.